site/pages/create-disruption.tsx

```diff
@@ -10,58 +11,195 @@ interface CreateDisruptionProps {
     inputs: DisruptionInfo;
 }
 
+enum Reason {
+    roadWorks = "Road Works",
+    vandalism = "Vandalism",
+    routeDiversion = "Route Diversion",
+    specialEvent = "Special Event",
+}
+
+const reasons: {
+    value: MiscellaneousReason | PersonnelReason | EnvironmentReason | EquipmentReason;
+    reason: string;
+}[] = [
+    {
+        value: MiscellaneousReason.roadWorks,
+        reason: "Road Works",
+    },
+    {
+        value: MiscellaneousReason.vandalism,
+        reason: "Vandalism",
+    },
+    {
+        value: MiscellaneousReason.routeDiversion,
+        reason: "Route Diversion",
+    },
+    {
+        value: MiscellaneousReason.specialEvent,
+        reason: "Special Event",
+    },
+];
+
+const getReasonOptions = (): JSX.Element[] => {
+    let options: JSX.Element[] = [];
```

Comment:
I think this can be a const - it only needs to be a let if we completely reassign the array.

Reply:
done